.gitlab-ci.yml

```diff
@@ -538,6 +573,7 @@ deploy_windows_testing:
 
 # run dd-agent-testing on windows
 kitchen_windows:
+  <<: *skip_on_cluster_agent_tag
```

Comment:
We skip stage `testkitchen_testing` and `testkitchen_cleanup` but not `testkitchen_deploy`. I think we should skip it too, especially since we're skipping the clean up.

Reply:
Since these jobs already have

```yaml
only:
    - triggers
```

they don't run when any tag is pushed so I don't think I need to add the `skip_on_cluster_agent_tag` here?